source/whats-new.txt

```diff
@@ -39,6 +41,19 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.0:
+
+What's New in 6.0
+-----------------
+
+The {+driver-short+} v6.0 release includes the following features:
+
+- This release removes support for the ``addUser()`` helper command. Use the
+  :manual:`createUser</reference/command/createUser>` command instead.
```

Comment:
Suggestion:

I think it could be helpful to provide information that the command the user should use is not a Node.js one. 

```suggestion
  :manual:`createUser </reference/command/createUser>` MongoDB Shell command instead.
```